src/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ foreach(domain ${TARGET_DOMAINS})
   add_subdirectory(${domain})
 endforeach()
 
+if (PORTBLAS_ENABLE_HALF)
+  # Set the variable used for C++ macro
+  set(ENABLE_PORTBLAS_HALF ON)
```

Comment:
According to #554 and PR #571 this var should be named with the prefix `ONEAPI_ONEMKL_`. Could you update it?
I know it would be the only one named correctly now, but I hope that PR will be merged soon.

Reply:
Yes I was planning to update this PR once https://github.com/oneapi-src/oneMKL/pull/571 is merged. It will be easier to do once I can add `ONEAPI_ONEMKL_ENABLE_PORTBLAS_HALF` to the list you introduce in https://github.com/oneapi-src/oneMKL/pull/571/files#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4R54